packages/client/src/components/tables/table-custom-columns-content.tsx

```diff
@@ -135,6 +138,19 @@ export const TableCustomColumnsContent = (props: TableCustomColumnsContentProps)
     // eslint-disable-next-line promise/prefer-await-to-callbacks
     callback(row);
   };
+  
+  const getEnumColumnValue = (c: TableColumn, uuid: string) => {
+    if(c.type === ColumnType.Enum) {
+      const enumConfig = c.config as { options: { id: string; label: string}[] };
+      const selectedOption = enumConfig.options.find(option => option.id == uuid); 
```

Comment:
Did you mean === ?

Reply:
yep! It checks equality and type.

Reply:
I was referring to 145, I noticed an option.id == uuid, thought you might have meant a === here